Modules/Scripted/DataProbe/DataProbe.py

```diff
@@ -69,6 +69,10 @@ def __init__(self, parent=None):
 
     self.frame = qt.QFrame(parent)
     self.frame.setLayout(qt.QVBoxLayout())
+    qSize = qt.QSizePolicy()
```

Comment:
```suggestion
    # Set horizontal policy to Ignored to prevent a long segment or volume name making the widget wider.
    # If the module panel made larger then the image viewers would move and the mouse pointer position
    # would change in the image, potentially pointing outside the node with the long name, resulting in the
    # module panel collapsing to the original size, causing an infinite oscillation.
    qSize = qt.QSizePolicy()
```